src/__tests__/functions.test.ts

```diff
@@ -369,6 +402,14 @@ describe('functions.test.ts', () => {
       const functions = require('../functions');
       await expect(functions.setSimulatedCard('_number')).resolves.toEqual({});
     });
+
+    it('collectInputs returns a proper value', async () => {
+      const functions = require('../functions');
+      await expect(functions.collectInputs()).resolves.toEqual({
```

Comment:
Can we create a test for the mapping methods separately? I can't clearly see what is the input we are passing into `mapFromCollectInputs`, and I'm unsure that the test that is written here is actually running the changed code.